CHANGES.md

```diff
@@ -19,6 +19,9 @@
   - `3DTILES_metadata`:
     - Added support for explicit content metadata. [#10150](https://github.com/CesiumGS/cesium/pull/10150)
 
+- Updated extensions to match the latest 3D Tiles 1.1 schema. See the [latest schema PR](https://github.com/CesiumGS/3d-tiles/pull/634):
+  - Folded the tileset-level metadata extension into the tileset schema. [#10153](https://github.com/CesiumGS/cesium/pull/10153)
```

Comment:
In the change log, I would be a little more explicit about the change, "folding" might not be obvious. Maybe

"tileset metadata is now stored in `tilesetJson.metadata` rather than the `3DTILES_metadata` extension"